docs/proposals/organizations.md

```diff
@@ -0,0 +1,137 @@
+# Organizations
+
+## Problems
+ 1.  Cluster administrators want to allocate a pool of resources to a subset of their company, and delegate the authority to manage how that pool is assigned to an organization administrator.
+ 2.  An organization administrator needs to be able to limit a particular project's usage of shared quota (think htb rate/ceil for tc qdisc classes).
+ 3.  Cluster administrators want to delegate control over groups of users to an organization administrator - that organization administrator should be able to manage the access of a set of users to individual projects under that organization umbrella.
+ 4.  Cluster administrators want to allow self-service of users on the cluster, but the total allocated resources those self service users can get access to is limited (to prevent abuse / unfair use of resources).
+
+We can do this by introducing the concept of project ownerhsip.  A Project is owned by at most one (Organization xor User).  We'll also introduce the idea of a `ClusterResourceQuota` object that can be associated with an organization or a user. An Organization can have multiple org-owners who are allowed to manage quota allocation to owned projects and OrgGroups (groups that scoped to projects owned by an Organization).
+
+### Open Questions
+ 3.  Can we punt on adding users as organization members?  Still don't see a clear answer on this.
+
+     Organization-admins should not be able to force users into an Org.  Users should not be able to join any Org they want.  I'd like to get the structure correct and punt on the ability to add members.  It would require some sort of invitation/request/approval flow.
+
+ 4.  Can we punt on project transfering?  Sounds like yes.  A cluster-admin could transfer ownership.
+
+     An org-owner shouldn't be able to take a project from someone else and they shouldn't be allowed to orphan their own project.  That means we'd need an offer/request/approval flow.
+
+ 6.  What things can org-owners do?
+
+ 	There's a list below proposing things.
+
+
+## ClusterResourceQuota
```

Comment:
Can you add a lot more detail to this section with maybe an end-to-end flow?

An example:
I have `ClusterResourceQuota` CQ.  I have a P1 and P2 `Project`.  Each project has a `ResourceQuota` RQ1 and RQ2.

Accounting:

Does CQ.Status.Used["cpu"] just take the sum of RQ1.Status.Used["cpu"] and RQ2.Status.Used["cpu"] or is it actually counting individual pods in P1 and P2 itself.  I think the former is a much more efficient model than the latter.

Basically, does CQ.Status.Used["cpu"] observe deletes and updates by just observing changes to the RQ1 and RQ2 entities its tracking?